hack/fetch-previous-tag.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+git fetch --tags
+git tag --sort version:refname | tail -2 | head -1 2>/dev/null
```

Comment:
this should work for now but have a look at https://gist.github.com/loisaidasam/b1e6879f3deb495c22cc where some issue with additional info set as part of the version may be present.

Reply:
Using now the better version as suggested in that gist.